pokerole_irl/user_app/models.py

```diff
@@ -11,3 +11,25 @@ class Profile(models.Model):
 
     def __str__(self):
         return self.user.username
+
+
+class Setting(models.Model):
+    owner = models.ForeignKey(User, on_delete=models.CASCADE)
+    members = models.ManyToManyField(User, related_name='+', blank=True)
+    name = models.CharField(max_length=30)
```

Comment:
A little stingy on the characters in my opinion, maybe 100 chars? But this can always be changed.

Reply:
Is now 100. Why not.